src/crawlee/browsers/_playwright_browser_controller.py

```diff
@@ -98,13 +98,12 @@ async def new_page(
         proxy_info: ProxyInfo | None = None,
     ) -> Page:
         if not self._browser_context:
-            self._browser_context = await self._create_browser_context(proxy_info)
+            self._browser_context = await self._create_browser_context(page_options, proxy_info)
 
         if not self.has_free_capacity:
             raise ValueError('Cannot open more pages in this browser.')
 
-        page_options = dict(page_options) if page_options else {}
-        page = await self._browser_context.new_page(**page_options)
+        page = await self._browser_context.new_page()
```

Comment:
Another question: Previously it was possible to pass arguments to self._browser_context.new_page(), now it is not possible. Was that not a valid use case?

Reply:
Earlier to create a page we used to use [`browser.new_page()`](https://playwright.dev/python/docs/api/class-browser#browser-new-page) method which takes page parameters. But some time ago we switched to using context and we create a page using [`browser_context.new_page()`](https://playwright.dev/python/docs/api/class-browsercontext#browser-context-new-page) which doesn't take any parameters. Now the page opening parameters must be set when creating the context